rvps/src/cache/local_fs/README.md

```diff
@@ -0,0 +1,15 @@
+# Local File System Cache
+
+This is a simple Cache, which will store the
+Reference Values in a local file. Thus the data
+of the RVPS is persistent, and also safe when it
+comes to restart or crash.
+
+The underlying storage engine is `sled`, which
```

Comment:
Have you considered using the [kv](https://docs.rs/kv/latest/kv/) crate instead?

Reply:
I have roughly investigated some key-value storages. Sled is the most popular one, [kv](https://docs.rs/kv/latest/kv/) is not used by many people. I'm not sure which is better, but I once used sled to develop a Key-Value Storage service and it worked well. So I choosed sled here.